samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj

```diff
@@ -76,6 +76,18 @@
     <ProjectReference Include="..\..\src\CommunityToolkit.Maui.Maps\CommunityToolkit.Maui.Maps.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
We don't need this section

Reply:
Thank you for pointing it out. Delete the above.